docs/how-tos/manage-fleet.md

```diff
@@ -25,69 +25,156 @@ For example, you might have 30 robots in one warehouse and 500 in another.
 You can monitor and teleoperate all of the robots from one online dashboard, and grant permission to other users to do the same.
 You can grant users different levels of access to individual machines or to groups of machines.
 
-If you'd like to follow a more detailed tutorial, see [Monitor Air Quality with a Fleet of Sensors](/tutorials/control/air-quality-fleet/).
+{{% alert title="In this page" color="info" %}}
 
-{{< table >}}
-{{% tablestep %}}
-**1. Create an account**
+1. [Organize your machines](#organize-your-machines): Learn about organizations and locations
+1. [Manage access with role-based access control](#manage-access-with-role-based-access-control)
+1. [Monitoring and debugging](/how-tos/manage-fleet/#monitoring-and-debugging)
+
+{{% /alert %}}
+
+## Prerequisites
+
+{{% expand "A Viam account" %}}
 
 Go to the [Viam app](https://app.viam.com) and sign up with Google, GitHub, Apple, or an email address.
 
-{{% /tablestep %}}
-{{% tablestep link="/cloud/" %}}
-**2. Create organizations and locations**
+{{% /expand%}}
+
+## Organize your machines
+
+Before you start connecting your devices to the Viam app, you need to decide how you want to group your devices.
```

Comment:
[opt] Maybe be more explicit about whether they can or cannot move machines and locations later?

Reply:
Adding but further down